### PR TITLE
Create and clean up dev command feature

### PR DESCRIPTION
**Pull request by rmenner**
## Summary by Sourcery

Implement a new 'dev' command using '@web/dev-server' with options for directory and port configuration, and enhance it with hot module replacement. Clean up the project by removing unused command files and updating dependencies.

New Features:
- Introduce a new 'dev' command using '@web/dev-server' with options to open a specific directory and change the server port.

Enhancements:
- Update the 'dev' command to use 'startDevServer' with hot module replacement (HMR) plugins for improved development workflow.

Chores:
- Remove unused command files 'default.js' and 'hello.js' from the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,33 +7,32 @@
     "auro": "./bin/index.js"
   },
   "scripts": {
-    "lint": "eslint --cache --fix && prettier . --write",
-    "prepare": "husky install"
+    "prepare": "husky"
```
[the remark this answers is not in the document]
Comment (rmenner) [reply package.json:10]: Husky install is deprecated

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1 @@
-#!/bin/sh
```
Comment (sun-mota) [inline .husky/pre-commit:1]: why do we need to remove this lines?

Comment (rmenner) [reply .husky/pre-commit:1]: The tutorial I initially used was not using the latest configuration best practices https://typicode.github.io/husky/how-to.html#adding-a-new-hook

File: web-dev-server.config.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { hmrPlugin, presets } from "@open-wc/dev-server-hmr";
 export default {
   plugins: [
     hmrPlugin({
-      include: ["src/**/*", "apiExamples/**/*"],
+      include: ["src/**/*", "demo/**/*", "apiExamples/**/*", "docs/**/*"],
```
Comment (jason-capsule42) [inline web-dev-server.config.js:6]: Should we just put the components directory in here now in anticipation of auro-form integration?

Comment (rmenner) [reply web-dev-server.config.js:6]: Since it runs from inside the component that folder is not necessary

**Comment by rmenner** [comment]
@sourcery-ai review

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**